{"uuid": "acefa0d7-4845-4d08-8490-85193817bb0d", "children": ["99d25419-dc14-4e36-ac60-080ba20837ac", "8fa43454-effd-4b55-9787-fa3023f111d8", "84c07c25-014e-4474-a515-d2d168756fd6"], "befores": [{"name": "_Class__pytest_setup_class", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '../api/api.yaml'\n", "trace": "  File \"/usr/local/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 703, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 634, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/dengying/PycharmProjects/gm/testcase/test_api.py\", line 13, in setup_class\n    cls.login = Testapi().login()\n  File \"/Users/dengying/PycharmProjects/gm/api/api_face.py\", line 16, in __init__\n    self.data = self.api_load(\"../api/api.yaml\")\n  File \"/Users/dengying/PycharmProjects/gm/api/base_api.py\", line 33, in api_load\n    return self.yaml_load(path)\n  File \"/Users/dengying/PycharmProjects/gm/api/base_api.py\", line 28, in yaml_load\n    with open(path) as f:\n"}, "start": 1588835843026, "stop": 1588835843026}], "start": 1588835843026, "stop": 1588835843145}